GitExtensions/Project.Publish.targets

```diff
@@ -268,7 +268,10 @@
     Creates an MSI.
     ============================================================
     -->
-  <Target Name="CreateMsi" AfterTargets="CreatePortable">
+  <PropertyGroup>
+    <Arm64Build>false</Arm64Build>
```

Comment:
Propose a rename to
```suggestion
    <IsArm64>false</IsArm64>
```

Reply:
I think a better name for the flag would be:
```
<IsArm64Build>false</IsArm64Build>
```
thus making it more meaningfully a `boolean` property.

Reply:
Works for me.